changelog/09-21-2023.md

```diff
@@ -26,9 +26,9 @@ In changelog we are introducing the first improvement to our closed alpha releas
 
 - Added ENV variables length validation, now it has a max of 150 characters
 - Added file size validation to 1GB per file
-- Fix for the redirects for invitations where users where not logged in 
-- Fix for the redirects for templates page where users where not logged in
-- Fix for the missing links in the navbar breadcrums
+- Fix for the redirects for invitations were users were not logged in 
```

Comment:
Fix for the redirects for invitations where users were not logged in.

missing h in first where